### ISSUE TITLE
Translations using wrong variable name cause error

### ISSUE THREAD

**Issue by nhannah** [connectedcars/react-i18n#29]
Currently t('This {word} is in a translated sentence.', { word: 'Variable'}) receiving the translation 'This {aWord} is in a translated sentence.' causes a crash. This leaves a fair deal of risk in the hands of the translator.

**Comment by mikfoo** [comment]
Good point, I've updated the title and also added another issue which is similar: https://github.com/connectedcars/react-i18n/issues/30

**Comment by miafoo** [comment]
We now have a eslint plugin to detect these things: https://github.com/connectedcars/eslint-plugin-react-i18n
